tikv/pd
# Redefine the total label query for balance leader and region schedulers

Signed-off-by: Hua Lu <hualu@gmail.com>



### What problem does this PR solve?
Issue Number: Close #5393 

### What is changed and how does it work?



```commit-message
* Remove the total label metric counters for leader and region
* Add new query in PD's grafana balance leader scheduler and balance region scheduler panels for the sum of all labels and name it as total
```

### Check List



Tests

- Manual test (add detailed scripts or steps below)
  - Build source code, copy over pd-server and pd.json in tiup directory
  - Start tiup playground and monitor the two scheduler panels
  - Verify the total label is the sum of all other labels
<img width="1734" alt="Screen Shot 2022-09-26 at 11 12 45 AM" src="https://user-images.githubusercontent.com/74505524/192316571-1baf36b7-3f52-4b50-a04a-144e120d4b1c.png">

Code changes


Side effects


Related changes


### Release note



```release-note
None
```
